provisioner_options.md

```diff
@@ -290,3 +291,13 @@ For example, the following suite will define grains on the guest:
 
 ### [vendor_path](id:vendor_path)
 
+### [additional_minion_config](id:additional_minion_config)
+The additional_)minion_config option allows you to specify additional options to set in the minion's config file.
```

Comment:
The additional_minion_config option allows you to specify additional options to set in the minion's config file.